test/spmd/tensor/test_dtensor_ops.py

```diff
@@ -668,7 +669,15 @@ def wrapped(fn):
 }
 
 
+skip_bw = [None, "torch.isfinite"]
+
+
 def run_dtensor_crossref(test_case, func, args, kwargs):
+    run_bw = False
+    if hasattr(args[0], "requires_grad") and resolve_name(func) not in skip_bw:
```

Comment:
if we always run backward, why we need the `run_bw` flag? or we want to skip some op where it first input is not a tensor?

Reply:
Thats right. I am following up on some of those cases as well to see if we need to support a list of input tensors from the ops test framework.